src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -304,16 +305,8 @@ export class PatientViewPageStore {
     readonly MDAndersonHeatMapAvailable = remoteData({
         await: () => [this.derivedPatientId],
         invoke: async() => {
-
-            let resp: any = await request.get(`//bioinformatics.mdanderson.org/dyce?app=chmdb&command=participant2maps&participant=${this.patientId}`);
-
-            const parsedResp: any = JSON.parse(resp.text);
-
-            // filecontent array is serialized :(
-            const fileContent: string[] = JSON.parse(parsedResp.fileContent);
-
+            const fileContent: string[] = await getHeatmapMeta(`//bioinformatics.mdanderson.org/participant2maps?participant=${this.patientId}`);

```

Comment:
can that service support both https and http?

Reply:
Yes